packages/php-wasm/compile/build-assets/php_wasm.c

```diff
@@ -412,6 +417,14 @@ void wasm_add_SERVER_entry(char *key, char *value) {
 	entry->value = strdup(value);
 	entry->next = wasm_server_context->server_array_entries;
 	wasm_server_context->server_array_entries = entry;
+
+	/**
+	 * Keep track of the document root separately so it can be reused
+	 * later to compute PHP_SELF.
+	 */
+	if( key == "DOCUMENT_ROOT" ) {
```

Comment:
`==` compares pointers, need to use `strcmp()` here. @wojtekn seems like `nx reset` got me and not you after all :-)

Reply:
fixed here https://github.com/WordPress/wordpress-playground/pull/280/commits/d0f26792c7e4b03a4d3111f3289f27218cdc74f0